### PR TITLE
Jump 020

### PR DESCRIPTION
**Pull request by matbesancon**
This bumps the JuMP version to 0.20, fixing some errors, we can either merge it or ignore it and merge one big PR with also the fixes for 0.21

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,3 +3,4 @@
 /dev/
 /docs/build/
 /docs/site/
+*.ipynb_checkpoints*
```
Comment (matbesancon) [inline .gitignore]:
```suggestion
*.ipynb_checkpoints*

```

File: script/modelling/power_systems.jl [hunk 2]
```diff
@@ -1,326 +1,329 @@
...
-    if status == MOI.OPTIMAL
-    push!(uc_df, (u_opt[1], u_opt[2], g_opt[1], g_opt[2], w_opt, ws_opt, obj))
-    else
-        println("Status: $status for demandscale = $demandscale \n")
-    end
-end
-
-#+
-
+#' ---
```
Comment (odow) [inline script/modelling/power_systems.jl]: Line endings in this file got borked?

**Review by odow — COMMENTED** [review]
Why not straight to 0.21?

File: script/modelling/power_systems.jl (hunk 2 above)
Comment (matbesancon) [reply script/modelling/power_systems.jl]: arg this one got me, thanks

**Comment by matbesancon** [comment]
> Why not straight to 0.21?

Indeed I considered it, just wanted to split PRs and keep them smaller, nothing makes grumpier than reviewing a PR for hours

File: script/modelling/power_systems.jl (hunk 2 above)
Comment (matbesancon) [reply script/modelling/power_systems.jl]: I'm not sure which tool is messing up with line endings

**Comment by matbesancon** [comment]
there was a version conflict for Julia 1.0 it seems, it will be a relief when the LTS bumps to something higher

**Comment by odow** [comment]
What's the version problem? Were you developing on Julia 1.3 or 1.4? The new artifact system is lower-bounded on 1.3.

**Review by odow — APPROVED** [review]

**Comment by odow** [comment]
Now I'm confused with these PRs. Should this be closed in favor of #56?

**Comment by matbesancon** [comment]
> What's the version problem? Were you developing on Julia 1.3 or 1.4?

Yes, I don't have any problem locally, but it occurs on travis

**Comment by odow** [comment]
Travis is running 1.0 and 1.1. If you use a manifest from 1.3, it's not guaranteed to work on 1.0.

**Comment by matbesancon** [comment]
> Now I'm confused with these PRs. Should this be closed in favor of #56?

It could, but I wanted to keep the PRs smaller to review, hence the incremental PRs

**Comment by matbesancon** [comment]
> Travis is running 1.0 and 1.1. If you use a manifest from 1.3, it's not guaranteed to work on 1.0.

yup learned it the hard way

**Pull request merged**